app/src/components/code/CodeEditorFormWrapper.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { ReactNode, useState } from "react";
+import { clsx as classNames } from "clsx";
+import { css } from "@emotion/react";
+
+import { Field, FieldProps, ValidationState } from "@arizeai/components";
+
+const codeEditorFormWrapperCSS = css`
+  &.is-hovered {
+    border: 1px solid var(--ac-global-input-field-border-color-active);
+  }
+  &.is-focused {
+    border: 1px solid var(--ac-global-input-field-border-color-active);
+  }
+  &.is-invalid {
+    border: 1px solid var(--ac-global-color-danger);
+  }
+  border-radius: var(--ac-global-rounding-medium);
+  border: 1px solid var(--ac-global-input-field-border-color);
+  width: 100%;
+  .cm-gutters,
+  .cm-content,
+  .cm-editor {
+    border-radius: var(--ac-global-rounding-medium);
+  }
+  .cm-focused {
+    outline: none;
+  }
+  transition: all 0.2s ease-in-out;
+`;
+
+/**
+ * Wrapper for code editor components (e.g. JSONEditor) that provides hover, focus, and validation state styles
+ */
+export function CodeEditorFormWrapper({
```

Comment:
went with children here and generic CodeEditorFormWrapper (instead of specifically wrapping the json editor. Since in theory this can and should be applied to any code editor in a form (may or may not always be json).